packages/acceptance-tests/pkg-tests-specs/sources/node-modules.test.ts

```diff
@@ -1855,6 +1855,49 @@ describe(`Node_Modules`, () => {
       }),
   );
 
+  it(`should work with user-created <workspace>/node_modules symlinks`,
+    makeTemporaryEnv(
+      {
+        workspaces: [`ws`],
+        dependencies: {
+        },
+      },
+      {
+        nodeLinker: `node-modules`,
+        nmHoistingLimits: `workspaces`,
+      },
+      async ({path, run}) => {
+        await xfs.mkdirpPromise(ppath.join(path, `ws`));
+        const trueInstallDir = ppath.resolve(path, `target`);
+        await xfs.mkdirPromise(trueInstallDir);
+
+        await xfs.mkdirpPromise(ppath.join(path, `target`));
+
+        await xfs.writeJsonPromise(ppath.join(path, `ws/${Filename.manifest}`), {
+          name: `ws`,
+          devDependencies: {
```

Comment:
does this behave the same if we use dependencies instead of devDependencies?

Reply:
Yep, they behave the same